integrations/doc_test.go

```diff
@@ -0,0 +1,372 @@
+package integrations
+
+import(
+	"fmt"
+	"context"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"log"
+	"go.mongodb.org/mongo-driver/bson"
+)
+func ExampleNewMongoCollection(){
+	clientOptions := options.Client().ApplyURI("mongodb+srv://<name>:<password>@<DB>.mujuh.mongodb.net/<collection>?retryWrites=true&w=majority")
```

Comment:
this shouldn't have mujuh. In fact you can just use localhost for example as here - https://github.com/keploy/test-app-url-shortner/blob/6227f1dc092540b10902b973e69c004e2e48e0da/main.go#L157